scripts/test-network-stalls.go

```diff
@@ -0,0 +1,225 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// +build ignore
+
+// Tests whether the uplink tool correctly times out when one of the storage nodes it's talking to
+// suddenly stops responding. In particular, this currently tests that happening during a Delete
+// operation, because that is where we have observed indefinite hangs before.
+
+package main
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/internal/memory"
+)
+
+var (
+	numTries      = flag.Int("num-tries", 20, "number of tries to cause a hang")
+	bucketName    = flag.String("bucket", "bukkit", "name of bucket to use for test")
+	fileSize      = flag.Int64("file-size", int64(5*memory.MiB), "size of test file to use")
```

Comment:
ah, what I meant was to use something like:

```
var filesize memory.Size = 5*memory.MiB
func init() {
    flag.Var(&filesize "file-size", "size of test file to use")
}
```

I know it's slightly annoying to write that way and probably doesn't matter that much here. But this allows to specify the command line flag as `command -file-size 5MB`

Reply:
oh, excellent. i didn't realize that was possible. i like it